pypesto/petab/importer.py

```diff
@@ -1,4 +1,6 @@
+from pypesto.objective.constants import MODE_FUN, MODE_RES
```

Comment:
```suggestion
from ..objective.constants import MODE_FUN, MODE_RES
```

Relative imports are a few lines below, could move this there.

Reply:
Cleaned up imports.